deploy/charts/templates/deployment.yaml

```diff
@@ -30,11 +30,11 @@ spec:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}" # In Jenkins, need to call helm with --set image.tag=master-XY
         command: ["/usr/bin/npm"]
-        args: ["start", "--", "--port=8888", "--config={{ .Values.deployenv.environment }}", "--prefix", "/ui/ui-build"]
+        args: ["start", "--", "--port={{ .Values.service.port }}", "--config={{ .Values.deployenv.environment }}", "--prefix", "/ui/ui-build"]
```

Comment:
One of my concerns with all this (and I'm fine rolling it out and seeing what happens at this stage) is whether the UI server is actually serving https. Our Kubernetes ingress model terminates ssl at the ingress controller, and cluster-internal traffic can be unencrypted. Thus if UI is trying to serve https itself, I think with the current configuration we're going to have some ssl issues. Might be nothing more than a certificate warning right now, and, as I mentioned, we can roll with this and see what happens. @mcstover Any thoughts on UI serving https by default?

Reply:
Is the UI trying to serve https?  My assumption was that it was not.

Reply:
I am basing this on vague recollection, but I think Matt will know immediately. In any event, the concern came up mainly while thinking about these ports, but it's not really germane to this PR. I would recommend we merge this in and continue revving on the build / deploy.

Reply:
All of the SSL resolution must be handled before entry into app.

Reply:
Ok, great. This is exactly what we want.